ovirt_imageio/_internal/backends/http.py

```diff
@@ -504,7 +505,8 @@ def _options(self):
         return options
 
     def _get_extents(self, context):
-        self._con.request("GET", self.url.path + "/extents?context=" + context)
```

Comment:
Are we testing extents with path=/?

This is wrong, the application serve only /images/ticket-id/extents.

Better fix the test to match the real code. This is not a library that need to handle any crazy input.

Regardless, joining without creating double // is good change. But the fact that you need to convert the joined path to string is worrying for code in the data path.

Reply:
> Are we testing extents with path=/?

Yes. And then the `Router` regex and the test `Handler` accounted for the extra `/`. I'll update the test as you suggest.

> Regardless, joining without creating double // is good change. But the fact that you need to convert the joined path to  string is worrying for code in the data path.

I agree in that it is a good change. I thought in doing the fix only in the test (since in a real scenario this should not happen), but in the end using `urljoin` sounds correct. Other tests use a more realistic url with `/images/ticket-id/extents`, so passing the pipelines should be safe enough.